src/jit/Backend.cpp

```diff
@@ -48,6 +48,13 @@ extern "C" {
 #error Unsupported architecture
 #endif
 
+#if (defined SLJIT_CONFIG_ARM_32 && SLJIT_CONFIG_ARM_32)
```

Comment:
I would put this in `#if SIMD` guarded code.

Reply:
There is a `#if (defined SLJIT_CONFIG_ARM && SLJIT_CONFIG_ARM)` code below, put this there